pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -82,6 +82,7 @@ public CmdTopics(PulsarAdmin admin) {
         jcommander.addCommand("expire-messages-all-subscriptions", new ExpireMessagesForAllSubscriptions());
 
         jcommander.addCommand("create-partitioned-topic", new CreatePartitionedCmd());
+        jcommander.addCommand("create-non-partitioned-topic", new CreateNonPartitionedCmd());
```

Comment:
what about just this for the cli?
`pulsar-admin topics create my-topic`

Reply:
Sure, I was just following the standard for create-partitioned-topic. But I guess we could change that.